src/import-csv.js

```diff
@@ -0,0 +1,56 @@
+import fs from 'fs';
+import Promise from 'bluebird';
+import CouchBulkImporter from 'couchbulkimporter';
+import parse from 'csv-parse';
+import BulkBadger from 'bulkbadger';
+
+export function cli (file, url, ...csvOptions) {
+  if (!file) {
+    const msg = [
+      'Usage:',
+      '',
+      'nmo import-csv [file] [couchdb-url] [...<csv options> <pairs>]',
+    ].join('\n');
+    const err = new Error(msg);
+    err.type = 'EUSAGE';
+
+    throw err;
+  }
+
+  return importcsv.apply(importcsv, [file, url, createOpts(csvOptions)]);
+}
+
+export function createOpts (args) {
+  return args.reduce((obj, cmd) => {
+    let [key, val=true] = cmd.split('=');
+    obj[key] = val;
+    return obj;
+  }, {});
+}
+
+export function importcsv (file, url, {delimiter= ',', columns= 'true'}) {
+  return new Promise((resolve, reject) => {
+    const input = fs.createReadStream(file);
```

Comment:
try to add an `on('error')` here to catch non-existing / readable files, sadly error handling in node is still somehow uncomfortable